index.bs

```diff
@@ -1410,10 +1410,12 @@ authenticator. Since all extensions are optional, this will not cause a function
 Extensions are identified by a string, chosen by the extension author. Extension identifiers should aim to be globally unique,
 e.g., by using reverse domain-name of the defining entity such as `com.example.webauthn.myextension`.
 
+Note: Use of dot-separated notation here does not imply an object hierarchy.
```

Comment:
Could we go further and just strip the dots out of all the examples and predefined extensions? We could advise camel-casing, e.g. WebAuthnGeo or webauthnTxauthSimple. That seems better than straddling two boats as we'd be doing here.